### PR TITLE
[WEB3-699] Fix "Write Contract" Tab in Blockscout Contract Details Page

### PR DESCRIPTION
**Pull request by jmontesdeocanuez**

### CODE REVIEW

File: ui/address/contract/ContractRead.tsx [hunk 1]
```diff
@@ -38,6 +38,8 @@ const ContractRead = ({ addressHash, isProxy, isCustomAbi }: Props) => {
     },
   });
 
+  const filteredData = data?.filter((item) => item.stateMutability === 'view') || [];
```
Comment (yurikoinaba) [inline ui/address/contract/ContractRead.tsx:41]: Add also the other conditions
`item.stateMutability === 'view' || item.stateMutability === 'pure' || item?.constant`

**Review by yurikoinaba — APPROVED** [review]

File: ui/address/contract/ContractRead.tsx [hunk 2]
```diff
@@ -38,6 +38,10 @@ const ContractRead = ({ addressHash, isProxy, isCustomAbi }: Props) => {
     },
   });
 
+  const filteredData = data?.filter((item) => (
+    item.stateMutability === 'view' || item?.constant
```
Comment (yurikoinaba) [inline ui/address/contract/ContractRead.tsx:42]: Sorry, one more condition `item.stateMutability === 'pure'`

File: ui/address/contract/ContractRead.tsx (hunk 1 above)
Comment (jmontesdeocanuez) [reply ui/address/contract/ContractRead.tsx]: @yurikoinaba added the constant one, but `pure` is not a possible state on the [type definition](https://github.com/HorizenLabs/blockscout-frontend/blob/4d9edb4ce43b2a8e1f44f2a1e5c421aecf77cf62/types/api/contract.ts#L4), I think they're always going to be `view`.

I checked the updated source code and it's the same.

File: ui/address/contract/ContractRead.tsx (hunk 2 above)
Comment (jmontesdeocanuez) [reply ui/address/contract/ContractRead.tsx:42]: @yurikoinaba added the constant one, but pure is not a possible state on the [type definition](https://github.com/HorizenLabs/blockscout-frontend/blob/4d9edb4ce43b2a8e1f44f2a1e5c421aecf77cf62/types/api/contract.ts#L4), I think they're always going to be view.

I checked the updated source code and it's the same.

Comment (yurikoinaba) [reply ui/address/contract/ContractRead.tsx:42]: 👍 Ok, let's make sure during testing

**Pull request merged**